lib/events/codes.go

```diff
@@ -262,6 +262,16 @@ var (
 		Name: SessionRejectedEvent,
 		Code: SessionRejectedCode,
 	}
+	// LockCreatedE is emitted when a lock is created/updated.
+	LockCreatedE = Event{
```

Comment:
Are LockCreatedE or LockDeletedE used? Why do we need those definitions? (I haven't looked at this code before, so I really don't know).

Reply:
You're right, these seem to be needed only for legacy events.